### ISSUE TITLE
Applet not working - extension.module.main is not a function

### ISSUE THREAD

**Issue by borfo** [shuairan/gTile#35]
Hi - Thanks for your work on this.

The Gtile Applet isn't working for me in Mint/Cinnamon 2.4.6 - the extension seems to load ok, but the applet fails.  Any idea what would be causing this or how I could fix it?  Here's the log:

```
info t=1969-12-31T19:00:00Z ================ DBus connection established ===============
info t=2015-04-02T07:34:33Z loading user theme: /usr/share//themes/Mint-X/cinnamon/cinnamon.css
info t=2015-04-02T07:34:33Z added icon directory: /usr/share//themes/Mint-X/cinnamon/
info t=2015-04-02T07:34:33Z Loaded extension gTile@shuairan in 27 ms
info t=2015-04-02T07:34:33Z loaded at Thu Apr 02 2015 07:34:33 GMT-0400 (EDT)
info t=2015-04-02T07:34:33Z Loaded applet menu@cinnamon.org in 293 ms
...
info t=2015-04-02T07:34:53Z    Resized mintupdate.py (20.799999237060547x20.799999237060547px)
error t=2015-04-02T07:35:13Z extension.module.main is not a function
trace t=2015-04-02T07:35:13Z 
<----------------
createApplet@/usr/share/cinnamon/js/ui/appletManager.js:334
addAppletToPanels@/usr/share/cinnamon/js/ui/appletManager.js:248
onEnabledAppletsChanged@/usr/share/cinnamon/js/ui/appletManager.js:187
---------------->
error t=2015-04-02T07:35:13Z [Extension "gTile@shuairan"]: Failed to evaluate 'main' function on applet: gTile@shuairan/25```

**Comment by shuairan** [comment]
The applet wasn't updated for a long time...

**Comment by shuairan** [comment]
close -> fixed in other repo 02bb21d71fdd4fe16fa2565e561cfe647c06a2c7
